lib/helpers/convertStatementTimestamp.js

```diff
@@ -0,0 +1,92 @@
+import {
+  map,
+  has,
+  get,
+  isArray,
+  isPlainObject,
+  isString,
+  identity,
+  mapValues,
+} from 'lodash';
+
+const createFilterParser = (key, parser) => (filter) => {
+  let newFilter;
+  if (has(filter, key)) {
+    try {
+      newFilter = parser(filter);
```

Comment:
Had to change this to not return the filter straight away, as we still needed it to go and map through the object

Reply:
This should probably be part of the isPlainObject block.